### DISCUSSION TITLE
case opening // drop posibility

### DISCUSSION THREAD

**Discussion by ghost** [danielkrupinski/Osiris#3276] [Ideas]
Can you please add or change the inventory config so that drops of every case have the same possibility as a real case opening?
Now it seems that chances for red and purple drops are much higher than normal cases. In addition, the floats of weapons are lower than usual, too.
There are some case opening statistics that you can refer to.
(I mean, if you want any skin, you can just add it to your inventory. Case opening should be nearly the same as a real case opening, isn't it? XD )
(PS: I'm still learning C#, and not able to handle this problem yet... If anyone can help me, that will be great! )

**Comment by aoqia194** [comment]
There is already code from an imgui based project.
The GitHub Repo is linked below if you would like to check it out.

https://github.com/B3akers/csgo-fake-opening
